python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -470,6 +472,76 @@ def run_optimize_model_gpu(repo_id,
     return result
 
 
+def run_ipex_int8(repo_id,
+                      local_model_hub,
+                      in_out_pairs,
+                      warm_up,
+                      num_trials,
+                      num_beams):
+    from transformers import AutoModel, AutoModelForCausalLM
+    from transformers import AutoTokenizer, GPTJForCausalLM, LlamaTokenizer
+    import intel_extension_for_pytorch as ipex
+    from torch.quantization import quantize_dynamic
+    model_path = get_model_path(repo_id, local_model_hub)
+    st = time.perf_counter()
+    if repo_id in ['THUDM/chatglm-6b', 'THUDM/chatglm2-6b']:
+        model = AutoModel.from_pretrained(model_path, trust_remote_code=True, use_cache=True)
+        tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+        model = quantize_dynamic(model, dtype=torch.qint8)
+    elif repo_id in LLAMA_IDS:
+        model = AutoModelForCausalLM.from_pretrained(model_path, trust_remote_code=True,
+                                                     use_cache=True)
+        tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
+        model = quantize_dynamic(model, dtype=torch.qint8)
```

Comment:
quantize_dynamic belongs to PyTorch.

Maybe we should use ipex_weight_only_quantization in https://github.com/intel/intel-extension-for-pytorch/blob/v2.1.0%2Bcpu/examples/cpu/inference/python/llm/single_instance/run_llama_quantization.py#L276C1-L307

Reply:
```python
model= transformers.AutoModelForCausalLM(model_name_or_path).eval()

qconfig = ipex.quantization.get_weight_only_quant_qconfig_mapping(
  weight_dtype=torch.qint8, # or torch.quint4x2
  lowp_mode=ipex.quantization.WoqLowpMode.NONE, # or FP16, BF16, INT8
)

checkpoint = None # optionally load int4 or int8 checkpoint
model = ipex.optimize_transformers(model, quantization_config=qconfig, low_precision_checkpoint=checkpoint)
```